client/client.go

```diff
@@ -445,7 +445,7 @@ func NewClientWithKeyspaceName(
 func (c *client) initRetry(f func(s string) error, str string) error {
 	var err error
 	for i := 0; i < c.option.maxRetryTimes; i++ {
-		if err = f(str); err == nil || strings.Contains(err.Error(), "ENTRY_NOT_FOUND") {
```

Comment:
why remove it?

Reply:
We should set the keyspace id to 0 if there is no meta.